webroot/panel/groups.php

```diff
@@ -84,39 +91,35 @@
 
 echo "<h5>Current Groups</h5>";
 
-if ($USER->isPI() && count($groups) == 1) {
-    echo "You are only a member of your own PI group. 
-    Navigate to the <a href='" . $CONFIG["site"]["prefix"] . "/panel/pi.php'>my users</a> page to see your group.";
-}
+// if ($USER->isPI() && count($groups) == 1) {
+//     echo "You are only a member of your own PI group.
+//     Navigate to the <a href='" . $CONFIG["site"]["prefix"] . "/panel/pi.php'>my users</a> page to see your group.";
+// }
 
 if (count($groups) == 0) {
-    echo "You are not a member of any groups. Request to join a PI using the button below, 
+    echo "You are not a member of any groups. Request to join a group using the button below, 
     or request your own PI account on the <a href='" . $CONFIG["site"]["prefix"] .
     "/panel/account.php'>account settings</a> page";
 }
 
 echo "<table>";
 
 foreach ($groups as $group) {
-    $owner = $group->getOwner();
-
-    if ($USER->getUID() == $owner->getUID()) {
-        continue;
-    }
-
-    echo "<tr class='expandable'>";
-    echo
-    "<td>
-    <button class='btnExpand'>&#9654;</button>" . $owner->getFirstname() . " " . $owner->getLastname() . "</td>";
-    echo "<td>" . $group->getPIUID() . "</td>";
-    echo "<td><a href='mailto:" . $owner->getMail() . "'>" . $owner->getMail() . "</a></td>";
+    echo "<tr class='expandable viewGroup'>";
+    echo "<td> 
+    <div class='type' style='border-radius: 5px; padding-left: 10px; 
+    color: white; padding-right: 10px; text-align: center; font-size: 12px; 
+    background-color: " . $group->getGroupTypeColor() . ";'>" . $group->getGroupTypeName() . "</div></td>";
+    echo "<td>" . $group->getGroupUID() . "</td>";
+    echo "<td> <button class='viewGroup'>View Group</button> </td>";
+    echo "<input type='hidden' name='pi' value='" . $group->getGroupUID() . "'>";
     echo
     "<td>
     <form action='' method='POST' 
-    onsubmit='return confirm(\"Are you sure you want to leave the PI group " . $group->getPIUID() . "?\")'>
+    onsubmit='return confirm(\"Are you sure you want to leave the group " . $group->getGroupUID() . "?\")'>
     <input type='hidden' name='form_name' value='removePIForm'>
-    <input type='hidden' name='pi' value='" . $group->getPIUID() . "'>
-    <input type='submit' value='Leave Group'>
+    <input type='hidden' name='pi' value='" . $group->getGroupUID() . "'>
+    <input type='submit' value='Leave Group' style='margin-top: 0px'>
```

Comment:
Pressing on the `Leave Group` button on the groups page initially shows a dialog but then immediately redirects to the group details page.